app/controllers/api/v1/labels_controller.rb

```diff
@@ -10,4 +10,10 @@ def index
       end
     render json: { labels: label_list }
   end
+
+  def show
+    label = Thought.label_counts.find_by(id: params[:id])
```

Comment:
Think you can use .find here instead, as .find will automatically find the id of an object, other than that everything looks great 👍